src/scss/patterns/_alignment.scss

```
@@ -1,21 +1,29 @@
+@use '../config';
+
 .contain {
   display: flow-root;
 }
 
 [class*='align-'] {
-  margin: var(--shim) var(--inline-margin, 0);
+  margin: var(--shim) var(--inline-margin-right, var(--inline-margin, 0)) var(--shim) var(--inline-margin-left, var(--inline-margin, 0));
```

```suggestion
  margin-block: var(--shim); 
  margin-inline-start: var(--inline-margin-start, var(--inline-margin, 0));
  margin-inline-end: var(--inline-margin-end, var(--inline-margin, 0));
```
A bit easier to follow? And not mixing logical/physical names?